src/__tests__/Storage.ts

```diff
@@ -1,7 +1,9 @@
 import { AvoStorage } from "../AvoStorage";
+import { defaultOptions } from "../__tests__/constants";
```

Comment:
Maybe create a few tests to verify that getItem, setItem and removeItem work with both options of shouldLog?